core/templates/domain/statistics/PlaythroughObjectFactorySpec.ts

```diff
@@ -195,4 +220,28 @@ describe('Playthrough Object Factory', () => {
       }]
     });
   });
+
+  it('should throw error on invalid backend dict', () => {
+    const playthroughDict = {
+      playthrough_id: 'playthroughId1',
+      exp_id: 'expId1',
+      exp_version: 1,
+      issue_type: 'InvalidType',
+      issue_customization_args: {
+        state_names: {
+          value: ['state1', 'state2']
+        }
+      },
+      actions: []
+    };
+
+    expect(() => {
+      // TS ignore is used because playthrough dict is assigned a invalid type
+      // to test errors.
+      // @ts-ignore
+      pof.createFromBackendDict(playthroughDict);
+    }).toThrowError(
```

Comment:
```suggestion
    // TS ignore is used because playthrough dict is assigned a invalid type to
    // test errors.
    // @ts-ignore
    expect(() => pof.createFromBackendDict(playthroughDict)).toThrowError(
```

Reply:
done